### PR TITLE
check by proper master names in deploy_masters step

### PR DESCRIPTION
**Pull request by karmab**
Signed-off-by: karmab <karimboumedhel@gmail.com>

### CODE REVIEW

**Comment by derekhiggins** [comment]
I got something similar in the dev1 branch here
https://github.com/openshift-metalkube/dev-scripts/pull/182

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/426/

File: 07_deploy_masters.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ source ocp_install_env.sh
 # for the final solution, but for now we'll prototype the workflow here
 export OS_TOKEN=fake-token
 export OS_URL=http://localhost:6385/
+export KUBECONFIG=ocp/auth/kubeconfig
```
Comment (hardys) [inline 07_deploy_masters.sh]: This isn't needed as it's already set via sourcing common.sh

**Comment by hardys** [comment]
> I got something similar in the dev1 branch here
> #182

@derekhiggins how should we proceed, there are several changes queued in #182, if that's ready should we merge it, or do you want to iterate on this and any other overlapping patches?

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/430/

**Comment by derekhiggins** [comment]
> > I got something similar in the dev1 branch here
> > #182
> 
> @derekhiggins how should we proceed, there are several changes queued in #182, if that's ready should we merge it, or do you want to iterate on this and any other overlapping patches?

thanks Karim, #182 has been merged now so I think this can be closed as it has also been handled there.

**Pull request closed without merging**